sig-contributor-experience/contribex-review-guidelines-on-test-infra.md

```diff
@@ -0,0 +1,123 @@
+# Guiding policy for test-infra changes requiring review by SIG Contributor Experience
+
+This document serves as a guiding policy to help identify which changes in the `git.k8s.io/test-infra/` repository require a review from the Technical Leads of SIG Contributor Experience.
+
+It also outlines areas that are outside the scope of SIG Contributor Experience's review.
+
+Note: This document is not a binding or an exhaustive list, but aims to provide general guidance.
+
+---
+
+## Index
+
+1. [When a Review from SIG Contributor Experience is Required](#when-a-review-from-sig-contributor-experience-is-required)
+2. [Recommended Areas for ContribEx Review](#recommended-areas-for-contribex-review)
+    1. [Changes to `config/plugins.yaml`](#changes-to-configpluginsyaml)
+        - [Plugins under ContribEx Review](#plugins-under-contribex-review)
+        - [Plugins Configured for Specific Orgs/Repos](#plugins-configured-for-specific-orgsrepos)
+        - [External Plugins Configured for Specific Orgs/Repos](#external-plugins-configured-for-specific-orgsrepos)
+    2. [Changes to `config/config.yaml`](#changes-to-configconfigyaml)
+    3. [Changes to `config/prow/OWNERS` File](#changes-to-configprowowners-file)
+    4. [Changes to Prow CI Jobs and Their OWNERS Files](#changes-to-prow-ci-jobs-and-their-owners-files)
+    5. [Changes to `Label_sync` Path](#changes-to-label_sync-path)
+3. [Out of Scope](#out-of-scope)
+
+---
+
+## When a Review from SIG Contributor Experience is Required
+
+A review from SIG Contributor Experience is necessary if any changes introduced by a pull request to the test-infra repo:
+
+- Introduce changes to how humans interact with Prow and its components (changes that alter contributor user experience)
+- Introduce changes to how bot interact with Prow? (TODO: verify!)
```

Comment:
Hmm, I'm not sure if this falls under the purview of ContribEx. It would help to narrow the scope down to aspects that end up being surfaced to contributors themselves. For example:
- If there is a change in how the bot handles webhooks, that's probably better handled by SIG Testing considering they own that code and have expertise in it.
- If there is a change that introduces retries, but as a side result of that, the bot ends up commenting more or less than it did before, then that is something that we should be chiming in on.

Reply:
> If there is a change that introduces retries, but as a side result of that, the bot ends up commenting more or less than it did before, then that is something that we should be chiming in on.

Presumably a change like this would only introduce more bot comments if there is a bug, so this change wouldn't need review either.

I would expect under the current unwritten policy that a change like "prow now automatically closes PRs without CLA after 5 days" to require contribex review _of the proposed workflow_ and any technical details to remain under whatever project implements it (so probably prow/plugins technical owners), while "prow now handles transient github failures better" to remain SIG Testing / Prow owners.